src/main/java/noel/tasks/Deadlines.java

```diff
@@ -0,0 +1,37 @@
+package noel.tasks;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+
+    final String taskChar = "[D]";
```

Comment:
Since this is a constant, I think you should name it as `TASKCHAR` or `EVENT_CHAR`